mmocr/utils/ocr.py

```diff
@@ -9,6 +10,7 @@
 from mmocr.apis.inference import model_inference
 from mmocr.core.visualize import det_recog_show_result
 from mmocr.datasets.pipelines.crop import crop_img
+from mmocr.utils.box_util import stitch_boxes_into_lines
```

Comment:
I originally didn't include that function because I personally find it badly designed and the result is clunky. I might submit a PR for a better solution when I'll have some spare time.

Reply:
Agree, it's an experimental utility and definitely needs a better design and implementation. We are open to better solutions at any time.